extensions/amp-story/1.0/amp-story.js

```diff
@@ -1405,6 +1421,33 @@ export class AmpStory extends AMP.BaseElement {
     return this.element.hasAttribute(Attributes.STANDALONE);
   }
 
+  /**
+   * Returns an array of the supported orientations configured by the publisher.
+   * Defaults to ['portrait'].
+   * @return {!Array<string>}
+   * @private
+   */
+  getSupportedOrientations_() {
+    if (this.supportedOrientations_) {
+      return this.supportedOrientations_;
+    }
+
+    const supportedOrientationsAttribute =
+        this.element.getAttribute(Attributes.SUPPORTED_ORIENTATIONS);
+
+    if (!supportedOrientationsAttribute) {
+      return ['portrait'];
```

Comment:
I have a preference for making constants out of `'portrait'` and `'landscape'`, so that we get compiler errors instead of bugs if we mistype them

Reply:
Done